InsertionSortProj/InsertionSort/InsertionSort.cs

```diff
@@ -0,0 +1,61 @@
+﻿namespace InsertionSortProj
+{
+    public class InsertionSort
+    {
+        private static void Main(string[] args)
+        {
+            Console.WriteLine("Insert array to sort");
+            try
+            {
+                var res = ConsoleArrayRead();
+                Sort(res);
+                Console.WriteLine("There is your sorted array");
+                Console.WriteLine("[{0}]", string.Join(", ", res));
+            }
+            catch (Exception ex)
+            {
+                switch (ex)
+                {
+                    case FormatException e:
+                        Console.WriteLine("Failed! Example of input: -1 0 1 2");
+                        break;
+                    case ArgumentException e:
+                        Console.WriteLine("Expected array of integers");
+                        break;
+                    default:
+                        Console.WriteLine("Unknown exception");
+                        break;
+                }
+            }
+        }
+
+        public static void Sort(int[] arr)
+        {
+            for (int sortedLen = 0; sortedLen < arr.Length; sortedLen++)
+            {
+                for (int k = 0; k < sortedLen; k++)
+                {
+                    int i = sortedLen - k;
+                    if (arr[i] < arr[i - 1])
+                    {
+                        (arr[i], arr[i-1]) = (arr[i-1], arr[i]);
```

Comment:
```suggestion
                        (arr[i], arr[i - 1]) = (arr[i - 1], arr[i]);
```
Бинарные операторы всегда отделяются пробелами.